all.jsonnet

```diff
@@ -7,7 +7,7 @@ local commonConfig = {
   config+:: {
     local cfg = self,
     namespace: 'thanos',
-    version: 'v0.10.1',
+    version: 'v0.12.0-rc.0',
```

Comment:
Curious what happened with 0.11 release? :)

Reply:
0.11 is in quarantine. :mask:

Reply:
I will address this before merging this. Thanks @lilic.